build/dependencies.props

```diff
@@ -2,10 +2,9 @@
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
     <BenchmarkDotNetVersion>0.10.3</BenchmarkDotNetVersion>
-    <CoreFxVersion>4.3.0</CoreFxVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <MoqVersion>4.7.1</MoqVersion>
-    <NETStandardImplicitPackageVersion>$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
```

Comment:
Just a question here, is `NETStandardImplicitPackageVersion` not necessary anymore? I see some differences in the `dependencies.props` files accross repos (for example here https://github.com/aspnet/HtmlAbstractions/pull/39/files).